src/js_native_api_v8.cc

```diff
@@ -1718,6 +1718,54 @@ napi_status NAPI_CDECL node_api_create_property_key_utf16(napi_env env,
   });
 }
 
+napi_status node_api_create_property_key_utf8(napi_env env,
+                                              const char* utf8name,
+                                              size_t length,
+                                              napi_value* result) {
+  if (env == nullptr || utf8name == nullptr || result == nullptr) {
+    return napi_invalid_arg;
+  }
+
+  v8::Isolate* isolate = reinterpret_cast<napi_env__*>(env)->isolate;
+  v8::HandleScope handle_scope(isolate);
+
+  v8::Local<v8::String> maybe_key;
+  if (!v8::String::NewFromUtf8(isolate,
```

Comment:
Please use `v8impl::NewString` as it is done for the `node_api_create_property_key_utf16`. The same is for the `node_api_create_property_key_latin1`

Reply:
ı used, thank you